README.md

```diff
@@ -1,27 +1,89 @@
 # @typescript/analyze-trace
-Tool for analyzing the output of `tsc --generateTrace` automatically, rather than following the steps [here](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).
+
+A tool for analyzing the output of `tsc --generateTrace` in a fast and digestible way (rather than more involved diagnostics [here](https://github.com/microsoft/TypeScript/wiki/Performance-Tracing).
 
 Note: The goal is to identify clear-cut hot-spots and provide enough context to extract a small repro.
 The repro can then be used as the basis of a bug report or a starting point for manual code inspection or profiling.
 
 ## Usage
 
-First, build your project with `tsc --generateTrace traceDir`.  This will create a new `traceDir` directory with paired trace and types files.
+The short version is to run these commands:
```

Comment:
"TL;DR"?

Reply:
I guess that's the TL;DR of this sentence :D

I always wonder if writing "TL;DR" in docs is a risk.